easybuild/easyconfigs/c/CVXPY/CVXPY-1.0.28-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,57 @@
+easyblock = 'PythonBundle'
+
+name = 'CVXPY'
+version = '1.0.28'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.cvxpy.org/'
+description = """ CVXPY is a Python-embedded modeling language for convex optimization problems.
+ It allows you to express your problem in a natural way that follows the math,
+ rather than in the restrictive standard form required by solvers. """
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('SWIG', '4.0.1'),
+]
+
+use_pip = True
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
+}
+
+exts_list = [
+    ('dill', '0.3.1.1', {
+        'checksums': ['42d8ef819367516592a825746a18073ced42ca169ab1f5f4044134703e7a049c'],
+    }),
+    ('multiprocess', '0.70.9', {
+        'checksums': ['9fd5bd990132da77e73dec6e9613408602a4612e1d73caf2e2b813d2b61508e5'],
+    }),
+    ('osqp', '0.6.1', {
+        'checksums': ['47b17996526d6ecdf35cfaead6e3e05d34bc2ad48bcb743153cefe555ecc0e8c'],
+    }),
+    ('ecos', '2.0.7.post1', {
+        'checksums': ['83e90f42b3f32e2a93f255c3cfad2da78dbd859119e93844c45d2fca20bdc758'],
+    }),
+    ('scs', '2.1.1-2', {
+        'checksums': ['f816cfe3d4b4cff3ac2b8b96588c5960ddd2a3dc946bda6b09db04e7bc6577f2'],
+    }),
+    ('cvxpy', version, {
+        'checksums': ['212cd483e9a0fbc91d593aa599eb065615b058189f66c7881521b63e6493307a'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(namelower)s'],
```

Comment:
@smoors This is the default for `PythonBundle`,  so you can remove this custom `sanity_check_paths`

Reply:
not exactly (I've added %(namelower)s), but fine for me.